### PR TITLE
(tlg2042 new texts) new OGL work

### LINKED ISSUE #2747: (urn:cts:greekLit:tlg2042.tlg086.1st1K-grc1) new edition
[urn.cts.greekLit.tlg2042.tlg086.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/11806209/urn.cts.greekLit.tlg2042.tlg086.1st1K-grc1.xml.zip)
See https://trello.com/c/q3FSRmPE/93-edit-analectasacrasp01pitrgoog for information on text and editors

### LINKED ISSUE #2748: (urn:cts:greekLit:tlg2042.tlg044.1st1K-grc1) new edition
See https://trello.com/c/q3FSRmPE/93-edit-analectasacrasp01pitrgoog for information on text and editors
[urn.cts.greekLit.tlg2042.tlg044.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/11806226/urn.cts.greekLit.tlg2042.tlg044.1st1K-grc1.xml.zip)

### PR DESCRIPTION
**Pull request by lcerrato**
fix #2747 
fix #2748 

Note that I think a large chunk of the Psalms is missing or was never captured.

### CODE REVIEW

File: data/tlg2042/tlg044/tlg2042.tlg044.1st1K-grc1.xml
Comment (AlisonBabeu) [inline data/tlg2042/tlg044/tlg2042.tlg044.1st1K-grc1.xml:1]: Only thing I noticed was that you encoded editor two separate ways in the structured bibliographic section.
` <editor><persName>Jean Baptiste Pitra</persName></editor>
                        <editor>Hermann Mutschmann</editor>`

File: data/tlg2042/tlg086/tlg2042.tlg086.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+                <availability><licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence></availability>
+                <date when="2023-06-20"/>
+            </publicationStmt>
+            
+            <sourceDesc>
+                <biblStruct>
+                    <monogr>
+                        <author>Origen</author>
+                        <title xml:lang="lat">Analecta sacra spicilegio solesmensi parata</title>
+                        <editor><persName>Jean Baptiste Pitra</persName></editor>
```
Comment (AlisonBabeu) [inline data/tlg2042/tlg086/tlg2042.tlg086.1st1K-grc1.xml:63]: Again, here editor encoded two different ways.

**Review by AlisonBabeu — APPROVED** [review]
I think these look good!

File: data/tlg0086/tlg048/tlg0086.tlg048.1st1K-grc1.xml
[the remark this answers is not in the document]
Comment (lcerrato) [reply data/tlg0086/tlg048/tlg0086.tlg048.1st1K-grc1.xml:1]: No, it's a correction. Charlotte was listed twice.

File: data/tlg2042/tlg086/tlg2042.tlg086.1st1K-grc1.xml (hunk 1 above)
Comment (lcerrato) [reply data/tlg2042/tlg086/tlg2042.tlg086.1st1K-grc1.xml:63]: This is how the file was encoded when I opened it plus probably copy/paste. Will fix.

**Comment by lcerrato** [comment]
@AlisonBabeu Can you double check the last commit here? I made requested changes.

**Review by AlisonBabeu — APPROVED** [review]
This looks good.

**Pull request merged**